src/components/v2v3/V2V3Project/ProjectActivity/hooks/ProjectActivity.ts

```diff
@@ -151,6 +170,14 @@ export function useProjectActivity({
         key: 'useAllowanceEvent',
         value: null, // Exclude all useAllowanceEvents, no UI support yet
       },
+      {
+        key: 'distributeToPayoutSplitEvent',
```

Comment:
adding some events we should exclude since we don't show these directly in project activity list